src/Eloquent/Concerns/InteractsWithRelations.php

```diff
@@ -78,6 +78,10 @@ protected function fillConnections(array $connections)
      */
     protected function connectBelongsToRelation(Relations\BelongsTo $relation, $id)
     {
+        if (! $id) {
+            return $relation->associate(null);
+        }
+
         $model = $relation->getRelated()->findOrFail($id);
         $relation->associate($model);
```

Comment:
For consistency you should return at all method exits

Reply:
Hmm, there are no other early returns?

Reply:
No, but now there is a return at 82, but no return at 86.